sites/svelte.dev/content/blog/2023-02-01-whats-new-in-svelte-february-2023.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "What's new in Svelte: February 2023"
+description: "Minor versions and major satisfaction"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Since last month, we've seen a couple of minor versions across Svelte and SvelteKit, a bunch of educational content to support the 1.0 release, as well as the [State of JS survey](https://stateofjs.com) results...
+
+Svelte, once again, increased in satisfaction and interest, so thanks to everyone who voted!
+
+Let's dive into the updates...
+
+## What's new in SvelteKit
+- The submitter's value will now be included automatically when progressively enhancing <form method="get"> (**1.0.3**, [#8273](https://github.com/sveltejs/kit/pull/8273))
+- Added an error in dev mode if global fetch is used with relative URL (**1.0.8**, [#8370](https://github.com/sveltejs/kit/pull/8370))
+- A warning will now indicated that hydration may break if comments are removed from HTML (**1.0.11**, [#8423](https://github.com/sveltejs/kit/pull/8423))
+- A warning will occur when of page options are used in `.svelte` files or a missing `<slot />` in layout is detected (**1.1.0**, [#8475](https://github.com/sveltejs/kit/pull/8475))
+- The new `text(...)` helper makes it easy to generate text responses (**1.2.0**, [#8371](https://github.com/sveltejs/kit/pull/8371))
+- The public env can now be accessed within app.html (**1.2.0**, [Docs](https://kit.svelte.dev/docs/project-structure#project-files-src), [#8449](https://github.com/sveltejs/kit/pull/8449))
+- An error will now be thrown if cookie exceeds size limit (**1.2.1**, [#8591](https://github.com/sveltejs/kit/pull/8591))
+
```

Comment:
https://github.com/sveltejs/kit/pull/8651 was marked as a bug fix, but could be worth including because the issue it fixes had 18 thumbs up. The change allows users to develop SvelteKit on remote IDEs